### PR TITLE
Rename ExcludedTag and move filtering from fetchers

### PR DESCRIPTION
**Pull request by HedicGuibert**
This renames the ExcludedTag entity to ConferenceFilter because it was confusing.
It also removes the filtering from the fetcher and puts it in the fetch command.

Refers https://github.com/jolicode/starfleet/issues/108

### CODE REVIEW

File: src/Command/FetchConferencesCommand.php [hunk 1]
```diff
@@ -197,4 +197,18 @@ protected function updateExistingConference(Conference $existingConference, Conf
 
         return $updated;
     }
+
+    private function shouldBeIgnored(Conference $conference): bool
+    {
+        foreach ($this->conferenceFilterRepository->findAll() as $conferenceFilter) {
```
Comment (welcoMattic) [inline src/Command/FetchConferencesCommand.php:203]: This `findAll` call will be repeated many time, as the `shouldBeIgnored` function is called inside 2 foreach loops. May be an initial call that store the result in a variable will be faster and avoir many DB queries?

File: src/Command/FetchConferencesCommand.php [hunk 2]
```diff
@@ -68,9 +68,9 @@ protected function execute(InputInterface $input, OutputInterface $output): int
                 continue;
             }
 
-            foreach ($fetcher->fetch() as $url => $conferences) {
+            foreach ($fetcher->fetch() as $conferences) {
                 foreach ($conferences as $conference) {
```
Comment (lyrixx) [inline src/Command/FetchConferencesCommand.php:72]: Let's get ride of this double foreach

File: src/Command/FetchConferencesCommand.php [hunk 3]
```diff
@@ -197,4 +197,18 @@ protected function updateExistingConference(Conference $existingConference, Conf
...
+    private function shouldBeIgnored(Conference $conference): bool
+    {
+        foreach ($this->conferenceFilterRepository->findAll() as $conferenceFilter) {
+            if (fnmatch($conferenceFilter->getName(), $conference->getName(), FNM_CASEFOLD)) {
+                $conference->setExcluded(true);
+
+                return true;
+            }
+        }
+        $conference->setExcluded(false);
```
Comment (lyrixx) [inline src/Command/FetchConferencesCommand.php:210]: Lets move excluded = false into the conference constructor

**Review by lyrixx — COMMENTED** [review]
Nice 👍🏼 
3 minors things to update and it'll be OK

File: src/Command/FetchConferencesCommand.php (hunk 3 above)
Comment (HedicGuibert) [reply src/Command/FetchConferencesCommand.php]: Actually excluded is false by default so I just removed this line

**Review by lyrixx — APPROVED** [review]
Very good 👏🏼  👏🏼  👏🏼

**Pull request merged**